decks/food.mdx

```diff
@@ -9,371 +9,526 @@ export const theme = customTheme;
-import DanDemoConcurrent from '../assets/dan-demo-concurrent.png';
+#### Dates, events, distance
 
-<img src={DanDemoConcurrent} />
+</L.Row>
 
----
+<Notes>
 
-<Background />
+- An agent wouldn’t know what values to extract from a given user’s input. 
+ This is where entities come into play. 
+ Any information in a sentence, critical to your business logic, will be an entity. 
+ This includes stuff like dates, distance, currency, etc. 
+ There are system entities, provided by DialogFlow for simple things like numbers and dates. 
+ And then there are developer defined entities. 
+ For example, “category”, for a bot about Pokemon! We’ll dive into how to make a custom developer entity further in the post.
+
+</Notes>
 
-### "Wait a minute... I already know about this! What is this talk really about?"
 
 ---
 
-<L.Column>
+<L.Row>
 
-# Concurrent Mode
+## Context
 
-## A chain of unfortunate consequences
+### A context-aware bot can remember things, and hold conversations
+ 
+<Notes>
 
-</L.Column>
+- Final concept before we can get started with coding is “Context”. 
+  This is what makes the bot truly conversational. 
+  A context-aware bot can remember things, and hold a conversation like humans do.
+  Consider the following conversation:
+
+</Notes>
+
+</L.Row>
 
 ---
 
 <Background />
 
-### How does React "delay" rendering?
+“Hey, are you coming for piano practice tonight?”  
+
+“Sorry, I’ve got dinner plans.”  
+
+“Okay, what about tomorrow night then?”  
+
+“That works!”
+
+<Notes>
+
+- Did you notice what just happened? The first question is straightforward to parse: The time is “tonight”, and the event, “piano practice”.
+ However, the second question, “Okay, what about tomorrow night then?” doesn’t specify anything about the actual event. 
+ It’s implied that we’re talking about “piano practice”. 
+ This sort of understanding comes naturally to us humans, but bots have to be explicitly programmed so that they understand the context across these sentences.
+
+</Notes>
 
 ---
 
 <L.Column>
 
-## Sources of Slowdown
+## Let’s get started!
 
-- Slow component render functions
-- Slow state setter or reducer updater functions
-- Slow effect functions
+### Make a Reddit bot that tells a joke or an interesting fact from the day’s top threads on specific subreddits
 
 </L.Column>
 
 ---
 
-import DeferredSetState from '../assets/deferred-setstate.svg';
+<Background />
+
+<L.Column sx={{ minHeight: '0' }}>
 
-<L.Row>
+## 1. Creating an Agent
+- Log in to the DialogFlow dashboard using your Google account.
+- Click on “Create Agent”
+- Enter the name of the agent, and hit “Create”.
 
-## Deferred setState.
+</L.Column>
 
-### Any state setter or dispatch can be delayed.
+---
 
-</L.Row>
+import chatbot1 from '../assets/1.png';
 
-<DeferredSetState />
+<img src={chatbot1} />
 
 ---
 
-import InterruptRendering from '../assets/interrupt-rendering.svg';
+<Background />
+<L.Column sx={{ minHeight: '0' }}>
 
-<L.Row>
+## 2. Setting up a “Welcome” Intent
```

Comment:
For each of these instructions, there should be an explanation on what exactly they are setting up and not just step by step instructions